### PR TITLE
chore: changing tabnet params for census

### PR DESCRIPTION
**Pull request by Hartorn**
**IMPORTANT: Please do not create a Pull Request without creating an issue first.**

*Any change needs to be discussed before proceeding. Failure to do so may result in the rejection of the pull request.*

**What kind of change does this PR introduce?**

**Does this PR introduce a breaking change?**

**What needs to be documented once your changes are merged?**

**Closing issues**

Put `closes #XXXX` in your comment to auto-close the issue that your PR fixes (if such).

### CODE REVIEW

File: census_example.ipynb [hunk 1]
```diff
@@ -179,8 +194,8 @@
     "clf.fit(\n",
     "    X_train=X_train, y_train=y_train,\n",
     "    X_valid=X_valid, y_valid=y_valid,\n",
-    "    max_epochs=1000, patience=50,\n",
-    "    batch_size=1024, virtual_batch_size=128\n",
+    "    max_epochs=1000, patience=10,\n",
```
Comment (eduardocarvp) [inline census_example.ipynb]: Why patience 10? I think convergence is sometimes slow and somewhat long ups and downs on the validation metric have been observed. Is the goal is to have a notebook that runs quicker than the previous one?

Comment (Hartorn) [reply census_example.ipynb]: @eduardocarvp Yep, goal was to have a better performance with a shorter time.

Feel free to test and propose changes if you feel like this is bad

Comment (Optimox) [reply census_example.ipynb]: We made some changes about which columns to consider as categorical and it improved quite significantly the results so I guess we're good now, I'm closing this!

**Comment by Optimox** [comment]
see my comment and feel free to reopen or create another one!

**Pull request closed without merging**